cfme/common/provider.py

```diff
@@ -136,6 +135,16 @@ def create(self, cancel=False, validate_credentials=True, check_existing=False,
         else:
             created = True
 
+            # Enable alerts collection before adding the provider to avoid missing active
```

Comment:
You're only enabling alerts if the alert type is Prometheus.   This comment should say something about that.   The question that I feel needs answered by the comment is why do we need to enable alerts collection when it is the alert type Prometheus, and why we don't need to do that when it is not.

Reply:
@jamesooden Thanks for your question, 

The only protocol that supports alerts for openshift is Prometheus. When we adding an alerts endpoint on CFME, CFME automatically starts to listen for OpenShift's alerts but if the profiles are not enabled CFME ignore any alerts. I opened a BZ regarding this illogic behavior but for now, this is the implementation (https://bugzilla.redhat.com/show_bug.cgi?id=1514950).

About the "why only for Prometheus":
It just want to make sure that the value of ```alerts_type``` is not ```Disable```

I will add a reference to the CFME docs about this limitation